client/containers/MainContainer.jsx

```diff
@@ -12,10 +12,14 @@ const mapStateToProps = (state) => ({
 
   //vendor stuff
   vendorItems: state.vendor.vendorItems,
+  productPrice: state.vendor.productPrice,
+  productDetails: state.vendor.productDetails,
   marketName: state.vendor.marketName,
   blogPost: state.vendor.blogPost,
-  vendorName: state.vendor.vendorName,
```

Comment:
need to re-add vendor name

Reply:
Added the vendorName to the props to dispatch and the prop drilling